### ISSUE TITLE
The results for traffic and weather in PatchTST do not match the original paper.

### ISSUE THREAD

**Issue by DDW12222AWWWWWWWWWW** [emadeldeen24/TSLANet#10]
The PatchTST/64 model yields excellent results on large datasets, with a 96-horizon forecast window error of 0.149 for Weather and 0.360 for Traffic. However, in your paper, the PatchTST model with a 512 look-back window predicts errors of 0.160 and 0.401 for the 96-horizon window, respectively.

**Comment by HYD-PG** [comment]
Yes, I have also noticed this issue, and I used the ETTh1 dataset for training and running, and found that the method in the paper has higher computational complexity and parameter size than Patchtst
